test/conflicting_deps/second_crate/BUILD

```diff
@@ -7,6 +7,6 @@ load(
 
 rust_library(
     name = "example_name_conflict",
-    srcs = ["src/lib.rs"],
+    srcs = ["lib.rs"],
```

Comment:
Side question: What are your personal feelings on ditching "src/" here? I've been thinking on how source files in rust sort of automatically "require" other local files to exist if they have module declarations, and I've been wondering if maybe we should take a more serious look at this at some point.

Initially, I was in a camp of "abandoning crate organization conventions", but I know repeatedly disregarding language conventions isn't really a convincing attitude, so I wanted to run through the thought exercise of what "embracing" those conventions might look like.

I'm interested to hear your thoughts, and we can move this into a proper issue if you'd like.

Reply:
I have mostly been writing BUILD per dir, and avoiding the extra noise of src/ and main/ dirs. 

I don't think src and main dirs work as well for company-sized or polyglot projects, but maven has precipitated a similar structure in enterprise java land. I have also seen enough crates set explicit values for lib / main paths that I don't feel guilty doing this.